rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -148,6 +161,13 @@ service Admin {
 void ping()
 
 /*
+ * Add the DB to the host, throw exception if it already exists
+ * and overwrite is false.
```

Comment:
can you update this comment to reflect the new code?
we always throw an exception if the db already exists.